tests/sqlparser_hive.rs

```diff
@@ -534,6 +534,20 @@ fn parse_use() {
     );
 }
 
+#[test]
```

Comment:
thinking we can move this test over to common? the current code isn't specific to hive and the feature/syntax seems popular enough that the parser can always accept it as it does today

Reply:
I understand the preference now is for the parser to err on the side of accepting queries even if the dialect doesn't actually support them right? If so, then yes, we can move the tests to common. The SHOW syntax is somewhat popular but it's not a standard.

Reply:
Yeah we have a few scenarios where the parser accepts input that isn't supported by the dialect. I figured here it could be easier to do so just as well, the alternative would be to add explicit methods to the dialect trait i.e `dialect.supports_show_databases()` `dialect.supports_show_schemas()` etc which I think would also be fine if that's preferrable from an impl pov

Reply:
Kept it at the generic level. I have some more SHOW statements for Snowflake to add, so we'll see if these challenge this decision.